solution/ui/regulations/eregs-vite/src/components/resources/SearchGovResults.vue

```diff
@@ -0,0 +1,168 @@
+<script setup>
+import {
+    formatDate,
+    locationLabel,
+    locationUrl,
+} from "../../utilities/filters";
+
+import ResultsItem from "@/components/search/ResultsItem.vue";
+import SupplementalContentObject from "eregsComponentLib/src/components/SupplementalContentObject.vue";
+
+defineProps({
+    base: {
+        type: String,
+        required: true,
+    },
+    partsLastUpdated: {
+        type: Object,
+        required: true,
+    },
+    partsList: {
+        type: Array,
+        required: true,
+    },
+    results: {
+        type: Array,
+        default: () => [],
+    },
+});
+
+const openChar = ""; // &#xe000;
+const closeChar = ""; // &#xe001;
+const highlightClass = "search-highlight";
+
+const needsBar = (item) => item.date && item.name;
+
+const getHighlightTerms = (string, startChar, stopChar) => {
+    // pure fns for composition
+    const splitString = (str, splitChar) => str.split(splitChar);
+    const getFirstItem = (arr) => arr.shift();
+    const removeFirstItem = (arr) => {
+        getFirstItem(arr);
+        return arr;
+    };
+    const mapFn = (fn, arr, args = []) => arr.map((item) => fn(item, ...args));
+    const uniqArr = (arr) => Array.from(new Set(arr));
+
+    return uniqArr(
+        mapFn(
+            getFirstItem,
+            mapFn(
+                splitString,
+                removeFirstItem(splitString(string, startChar)),
+                [stopChar]
+            )
+        )
+    );
+};
+
+const formatLinkTitle = ({ description, descriptionHeadline, snippet }) => {
+    const termsToHighlight = getHighlightTerms(snippet, openChar, closeChar);
+
+    let linkTitle = descriptionHeadline || description;
+    termsToHighlight.forEach((term) => {
```

Comment:
bolded search term matches are not included in the `description` in the response from the API, but the story AC mandates that the `description` must included the same bolded search terms as the `snippet`.  I'm doing that on the front end here -- collecting the highlighted search terms from the `snippet` and then finding them in the `description` and surrounding them with a `<span>` tag that includes a bolded style class.